index.html

```diff
@@ -48,32 +48,71 @@ <h2>Advancing the future of the internet</h2>
         <!-- Benefits section -->
         <section class="mzp-l-content benefits-section">
             <div class="benefits-text-wrapper">
-                <h2><span class="dark-highlight">Benefits & Perks</span></h2>
+                <h2><span class="dark-highlight">Why Join Us?</span></h2>
             </div>
-            <div class="mzp-t-picto-center">
+            <div class="why-join-container">
+                <h3 class="why-join-subheader">For Builders</h3>
```

Comment:
**Nit:** Shouldn't `For Sponsors` be first, and `For Builders` be third in the section's columns?

Reply:
Uhm, not from what I can tell. If this needs to be changed later on, happy to do a follow up PR

Reply:
The Figma design shows otherwise, so I was referring to that